ims-examples/ims-1-remote.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset='utf-8'>
+  <script src='https://purl.imsglobal.org/spec/respec-ims-default.js' async class='remove'></script>
```

Comment:
Should this be a relatively sourced file? If not, perhaps the purl location needs to be updated (if it hasn't already)?